code/datums/gamemodes/nuclear_war.dm

```diff
@@ -37,6 +37,11 @@
 		/datum/job/xenomorph = NUCLEAR_WAR_LARVA_POINTS_NEEDED,
 	)
 
+	evo_requirements = list(
+		/datum/xeno_caste/king = 12,
+		/datum/xeno_caste/queen = 4,
```

Comment:
Uhhh ???

Reply:
Nukewar was to be given Queen at 4 pop as discussed in #walance without objections at the time and now with approval of Kurt.
I'll change the number to 8 for now to get the crash changes in and make a separate pr for changing the queen value to 8 if you want.

Reply:
I don't see in what world where queen is gonna be balanced at 4 pop hive

Reply:
Alright, removed.